code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm

```diff
@@ -1,45 +1,37 @@
 /obj/item/bodypart/head/moth
-	icon = 'icons/mob/species/moth/bodyparts.dmi'
-	icon_state = "moth_head"
-	icon_static = 'icons/mob/species/moth/bodyparts.dmi'
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/moth_parts_greyscale.dmi'
```

Comment:
add skyrat edit comments to all of this.

Reply:
Done. Honestly, I didn't even begin to think TG had moths. My bad